<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string>测试文本
测试文本测试文本
测试文本
测试文本
测试文本
测试文本
测试文本
测试文本
测试文本测试文本测试文本测试文本测试文本
测试文本测试文本测试文本
测试文本</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actFile_new"/>
    <addaction name="actFile_open"/>
    <addaction name="actFile_save"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actEdit_cut"/>
    <addaction name="actEdit_copy"/>
    <addaction name="actEdit_paste"/>
    <addaction name="separator"/>
    <addaction name="actEdit_undo"/>
    <addaction name="actEdit_redo"/>
    <addaction name="separator"/>
    <addaction name="actEdit_selectAll"/>
    <addaction name="actEdit_clear"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>格式(&amp;M)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>界面语言</string>
     </property>
     <addaction name="actLang_CN"/>
     <addaction name="actLang_EN"/>
    </widget>
    <addaction name="actFont_bold"/>
    <addaction name="actFont_italic"/>
    <addaction name="actFont_underline"/>
    <addaction name="separator"/>
    <addaction name="actSys_toggleText"/>
    <addaction name="menu"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_M"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_new"/>
   <addaction name="actFile_open"/>
   <addaction name="actFile_save"/>
   <addaction name="separator"/>
   <addaction name="actEdit_cut"/>
   <addaction name="actEdit_copy"/>
   <addaction name="actEdit_paste"/>
   <addaction name="separator"/>
   <addaction name="actEdit_undo"/>
   <addaction name="actEdit_redo"/>
   <addaction name="separator"/>
   <addaction name="actFont_bold"/>
   <addaction name="actFont_italic"/>
   <addaction name="actFont_underline"/>
   <addaction name="separator"/>
   <addaction name="actLang_CN"/>
   <addaction name="actLang_EN"/>
   <addaction name="separator"/>
  </widget>
  <action name="actEdit_cut">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/200.bmp</normaloff>:/Icon/icons/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actEdit_copy">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/202.bmp</normaloff>:/Icon/icons/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actEdit_paste">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/204.bmp</normaloff>:/Icon/icons/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>从粘贴板粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actFont_bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/500.bmp</normaloff>:/Icon/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actFont_italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/502.bmp</normaloff>:/Icon/icons/502.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actFont_underline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/504.bmp</normaloff>:/Icon/icons/504.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/132.bmp</normaloff>:/Icon/icons/132.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭本窗口</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actEdit_undo">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/206.bmp</normaloff>:/Icon/icons/206.bmp</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销上次编辑操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actEdit_redo">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/208.bmp</normaloff>:/Icon/icons/208.bmp</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做上次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actFile_new">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/100.bmp</normaloff>:/Icon/icons/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actSys_toggleText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/510.bmp</normaloff>icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>显示按钮文字</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏按钮文字</string>
   </property>
  </action>
  <action name="actFile_open">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/001.GIF</normaloff>:/Icon/icons/001.GIF</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actFile_save">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/104.bmp</normaloff>:/Icon/icons/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actLang_CN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/CN.jpg</normaloff>:/Icon/icons/CN.jpg</iconset>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
   <property name="toolTip">
    <string>中文界面</string>
   </property>
  </action>
  <action name="actLang_EN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/Icon/icons/timg2.jpg</normaloff>:/Icon/icons/timg2.jpg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="toolTip">
    <string>English Interface</string>
   </property>
  </action>
  <action name="actEdit_selectAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actEdit_clear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_copy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_cut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_paste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_redo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_undo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_selectAll</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
